### PR TITLE
Mobile-pagination styles

### PR DESCRIPTION
**Pull request by d2bit**

### CODE REVIEW

**Comment by scruti** [comment]
:+1:

File: app/assets/stylesheets/psique/structures/_pagination.scss [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+// -------------------------------------
+//   Base
+// -------------------------------------
+
+// -------------------------------------
+//   Scaffolding
+// -------------------------------------
+
+// ----- disabled ----- //
+.mobile-pagination .disabled { color: $mobile-pagination-disabled-color; }
```
Comment (gponsu) [inline app/assets/stylesheets/psique/structures/_pagination.scss]: I think that 'disabled' is not a scaffolding element but an state:

```sass
// -------------------------------------
//   States
// -------------------------------------

// ----- Is disabled ----- //

.is-disabled {
  .mobile-pagination & {  color: $mobile-pagination-disabled-color; }
}
```

Comment (gponsu) [inline app/assets/stylesheets/psique/structures/_pagination.scss]: I think that 'disabled' is not a scaffolding element but an state:

```sass
// -------------------------------------
//   States
// -------------------------------------

// ----- Is disabled ----- //

.is-disabled {
  .mobile-pagination & {  color: $mobile-pagination-disabled-color; }
}
```

**Comment by Fernan2** [comment]
+1

**Comment by gponsu** [comment]
:-1:

**Pull request merged**